users/service.go

```diff
@@ -558,7 +563,7 @@ func (svc *service) checkSuperAdmin(ctx context.Context, adminID string) error {
 		if err := svc.clients.CheckSuperAdmin(ctx, adminID); err != nil {
 			return errors.Wrap(svcerr.ErrAuthorization, err)
 		}
-		return errors.Wrap(svcerr.ErrAuthorization, err)
+		return nil
```

Comment:
@JeffMboya   Could please explain the reason for this change. 
This should not be like this, If the user is not SuperAdmin in policies, then it should send Authorization error